tools/cloud-build/daily-tests/ansible_playbooks/spack.yml

```diff
@@ -14,18 +14,11 @@
 
 ---
 
-- name: Wait for startup script to complete
-  become: true
-  wait_for:
-    path: /var/log/messages
-    search_regex: '.*{{ login_node }}.*startup-script exit status ([0-9]+)'
-    timeout: 7200
-    state: present
-  register: startup_status
-- name: Fail if startup script exited with a non-zero return code
-  fail:
-    msg: There was a failure in the startup script
-  when: startup_status['match_groups'][0] != "0"
+- name: Include wait for startup script
```

Comment:
I'm curious if you could do the same thing with a role, similar to this: https://github.com/dellhpc/omnia/blob/v1.2.2/omnia.yml#L20

I think you just need to create a "roles" directory with that name in it, then you can run multiple roles with one task. I wouldn't put much time into it, but if it's easy it could be worth it and decrease the boilerplate.